### PR TITLE
Add events

### PR DESCRIPTION
**Pull request by canyon289**

### CODE REVIEW

**Comment by purna135** [comment]
Hi @canyon289, thank you so much for the PR, and I apologise for missing it; I'm not sure why I didn't receive notification of this PR.

One question: is this real event data or a dummy event to be added, as the date and social links are out of date?

**Comment by canyon289** [comment]
yes but let's merge this first so we can see if everything shows up and I'll make it correct in a second pr

**Comment by purna135** [comment]
Ok

File: data/events.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #==============================================================
 
 #[[event]]
-#    dataTime = "Thu Nov 17, 2022, 8:30 PM IST"      (Any data and time format)
+#    dataTime = "Feb 9th, 2022"          (Any data and time format)
```
Comment (canyon289) [inline data/events.toml:6]: Updated for the merge

**Pull request merged**